CHANGELOG.md

```diff
@@ -2,510 +2,321 @@
 
 ## Unreleased 3.0.0 (Breaking release)
 
-See the [versioning documentation for how to update this
-changelog](./docs/contributing/versioning.md#updating-changelog).
+## Breaking changes
+You must make the following changes when you migrate to this release, or your service may break.
 
-- Renamed css class names `js-character-count` and `js-header-toggle`
+### Update file paths, attributes and class names
+To make sure your project's files do not conflict with GOV.UK Frontend, we've moved our package files into a directory called `govuk`.
 
-  Fixes issues with govuk-fronted javascript clashing with govuk-template/govuk-element and potentially other third-party javascript libraries using `js-` class names as hooks.
+#### If you’re using Sass
```

Comment:
Could we say "GOV.UK Frontend Sass".

Reply:
I've added it to a list of potential post-release style changes.